src/main/groovy/com/getkeepsafe/dexcount/DexFile.groovy

```diff
@@ -80,16 +80,24 @@ class DexFile {
             }
         }
         // convert it to DEX format by using the Android dx tool
-        def androidSdkHome = System.getenv("ANDROID_HOME")
+        def androidSdkHome = DexMethodCountPlugin.sdkLocation
         if (androidSdkHome == null) {
             throw new Exception("ANDROID_HOME env variable not defined!")
```

Comment:
the SDK location is only needed if you process an AAR file...so that's why it's here instead...I'll update the msg